src/styles/index.scss

```diff
@@ -11,4 +11,6 @@
 @import "./components/webchatx.scss";
 
 // pages
-@import "./pages/login.scss";
+
+// components
+@import "./components/webchat";
```

Comment:
Move up with other components in this file.

Reply:
Got rid of this line wasn't needed